ntoskrnl/ex/uuid.c

```diff
@@ -400,9 +400,10 @@ ExUuidCreate(OUT UUID *Uuid)
         {
             Time.QuadPart = ExpUuidCachedValues.Time;
 
+            C_ASSERT(sizeof(UUID_CACHED_VALUES_STRUCT) - FIELD_OFFSET(UUID_CACHED_VALUES_STRUCT, ClockSeqHiAndReserved) >= sizeof(Uuid->Data4));
```

Comment:
I don't like the assertion placement. Can we move it to the GLOBALS section on the top and put some comment along with it?

Reply:
I have moved assertion to global.